content/docs/start/data-and-model-versioning.md

```diff
@@ -48,10 +48,10 @@ $ dvc add data/data.xml
 ```
 
 DVC stores information about the added file (or a directory) in a special `.dvc`
-file named `data/data.xml.dvc`, a small text file with a human-readable
-[format](/doc/user-guide/project-structure/dvc-files). This file can be easily
-versioned like source code with Git, as a placeholder for the original data
-(which gets listed in `.gitignore`):
+file named `data/data.xml.dvc` - a small text file with a human-readable
+[format](/doc/user-guide/project-structure/dvc-files). This metadata file can be
+easily versioned like source code with Git. The original data, meanwhile, is
+listed in `.gitignore`:
```

Comment:
Good rephrase, but "as a **placeholder** for the original data" was a key idea.

Reply:
Also ``is listed in `.gitignore`:`` looks like we are going to demo the .gitignore contents below (which we don't do).

Reply:
reworded